Dangerfile

```diff
@@ -32,11 +29,21 @@ reviewers = requested_reviewers + actual_reviewers
 pr_reviewers = reviewers.map { |u| u['login'] }
 # Official reviewer
 official_reviewer = %w[mpermperpisang mmpisang mpermper321]
+# Add reviewer based on file contributor
+file_changed_list = github.api.pull_request_files(repo, pr_num)
+file_changed_name = file_changed_list.map { |u| u['filename'] }
+committer_list = file_changed_name.map { |u| message u['filename'].to_s }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mpermperpisang/golang-automation/blob/fca08e1e08a42d38ae41be50ddb33dee99b092fd/Dangerfile#L35"></span>Useless assignment to variable - <code>committer_list</code>. Did you mean <code>committer_user</code>?</td>
    </tr>
  </tbody>
</table>